include/ffi.h.in

```diff
@@ -227,6 +227,9 @@ typedef struct {
   ffi_type *rtype;
   unsigned bytes;
   unsigned flags;
+#ifdef _M_ARM64
+  unsigned isVariadic;
```

Comment:
Two nits here.  I think the general style of libffi is to use snake case, so this should be `is_variadic`.  Also, `FFI_EXTRA_CIF_FIELDS` exists so that ports can add extra fields here, without needing to `#if` the generic code.  So, I think this ought to go in `src/aarch64/ffitarget.h` somewhere instead.

Actually, I don't see where this field is used, either.  If it's in the assembly (I don't know aarch64 assembly) then maybe a comment there would be good.

Reply:
isVariadic is used in ffi.c and prep_cif.c
It was added in #486 so those changes weren't showing in the diffs for this PR
In #486 isVariadic was only added to  msvc_build/aarch64/aarch64_include/ffi.h and not ffi.h.in
I renamed isVariadic to is_variadic moved itto ffitarget.h using the FFI_EXTRA_CIF_FIELDS define in the next commit.